common/app/contentapi/ContentApiClient.scala

```diff
@@ -76,7 +76,7 @@ trait ApiQueryDefaults extends GuLogging {
 
   def item(id: String): ItemQuery = CapiContentApiClient.item(id)
 
-  def item(id: String, edition: Edition): ItemQuery = item(id, edition.id)
+  def item(id: String, edition: Edition): ItemQuery = item(id, EditionIdCAPIMapper.mapEditionId(edition))
```

Comment:
The mapping looks good to me as Content-api's Conceirge reads `us, uk, au, europe, international` so it has to be this to give the content. 
Mapping done in this PR should be fine. Still will get CoPip team to give review on this as well. 

Just being curious, some que on this:
1. Can we try changing Edition id from `EUR` to `EUROPE`?https://github.com/guardian/frontend/blob/85fccaf6d3489c972437afe5a28074e5733eece8/common/app/common/editions/Europe.scala#L10 

3. If not above, then can we try accessing networkfrontId here as that refers to `europe`, so can we try checking if edition.id != edition.networkfrontId (which is true in this case) so then access `networkFrontId` instead? (I know here priority should be `edition.id` but just curious to know if we can do these cases, then we might not need mapping.

Reply:
Thanks @ioannakok for your answers.
Details about the work it requires to refactor in many places, I think it does help in justifying to go ahead with the mapping at the moment in this PR as a best solution.
Indeed we would want to keep a consistent name (example "EUROPE") as id across all Guardian repos hence #26851 would be a great idea. Thanks for that, Ioanna.
We are happy to approve this.

Reply:
Regarding your query:
_Is Concierge the interface to all CAPI requests? Does that mean that all of CAPI's endpoints expect the long form id i.e. europe, international?_

Yes
(but I will confirm with my team as well) 👍

Reply:
Team also has confirmed the same for Concierge 👍 
As aforementioned, consistent name would be great idea to keep and as long as Concierge reading correct form of `id` it should return the content whether any long form in it. Thankyou.

Reply:
Hi @Divs-B thanks for your review!

Out of an abundance of caution we checked in Kibana whether any other requests (other than most-viewed) are made to CAPI from frontend using the short edition ids of `EUR` or `INT`. Using the query in the `Content Platforms` space and `logstash-capi-*` index we could see a variety of requests using `EUR` or `INT`

Here are few example queries to demonstrate the different frontend applications making requests to CAPI using the short form ids of either `EUR` or `INT`:

[EUR form the facia app](https://logs.gutools.co.uk/s/content-platforms/goto/6fb7f240-bc6c-11ee-af37-dd17d2a74de1)
[EUR from the onwards app](https://logs.gutools.co.uk/s/content-platforms/goto/4787dc40-bc6c-11ee-a620-1390ad1b8a15)
[INT from the article app](https://logs.gutools.co.uk/s/content-platforms/goto/87b72280-bc6c-11ee-a620-1390ad1b8a15)
[INT from applications](https://logs.gutools.co.uk/s/content-platforms/goto/473c85f0-bc6d-11ee-af37-dd17d2a74de1)
[INT from applications-crosswords](https://logs.gutools.co.uk/s/content-platforms/goto/25a96660-bc6d-11ee-ad1f-3b3933c96f60)

Manually updating the edition id to the long form id returns the same response so we assume the edition parameter does not actually affect the query response.

Could you please confirm our understanding that the queries above do not use the edition parameter so updating the edition id to the long form should have no effect.

Thanks!

Reply:
Hello. CAPI only becomes interested in the edition parameter if the query is for what looks like a network front (from days before the Fronts API existed IIRC) or an editionalised section, e.g. sport. In the examples you've linked, these appear to be mostly individual content item or tag searches, and those will ignore the edition. If you're curious about which sections are editionalised, you can go to `content.guardianapis.com/sections` and look for any with multiple editions listed. Hope this helps 🤞

Reply:
Thanks @JustinPinner and @Divs-B very helpful